genai-perf/genai_perf/logging.py

```diff
@@ -43,82 +73,42 @@ def init_logging() -> None:
         },
         "handlers": {
             "console": {
-                "level": "INFO",
+                "class": MOCK_HANDLER if is_testing else HANDLER,
                 "formatter": "standard",
-                "class": "logging.StreamHandler",
-                "stream": "ext://sys.stdout",  # Default is stderr
+                "highlighter": NullHighlighter(),
+                "omit_repeated_times": False,
+                "rich_tracebacks": True,
+                "show_path": True,
             },
         },
         "loggers": {
-            "": {  # root logger - avoid using
+            "": {  # root logger
                 "handlers": ["console"],
                 "level": "WARNING",
                 "propagate": False,
             },
             "__main__": {  # if __name__ == '__main__'
                 "handlers": ["console"],
-                "level": "DEBUG",
-                "propagate": False,
-            },
-            "genai_perf.parser": {  # must use module name for loggers
-                "handlers": ["console"],
-                "level": "DEBUG",
-                "propagate": False,
-            },
-            "genai_perf.wrapper": {
-                "handlers": ["console"],
-                "level": "DEBUG",
-                "propagate": False,
-            },
-            "genai_perf.plots.plot_config_parser": {
-                "handlers": ["console"],
-                "level": "DEBUG",
-                "propagate": False,
-            },
-            "genai_perf.plots.plot_manager": {
-                "handlers": ["console"],
-                "level": "DEBUG",
+                "level": log_level,
                 "propagate": False,
             },
-            "genai_perf.export_data.json_exporter": {
+            "genai_perf": {  # All modules in genai_perf package
```

Comment:
if we define a more specific handler, will it overwrite the general one?

Reply:
Not sure if I fully understood the question, but if we define a different handler, then we can update the `HANDLER` variable to point to it. (So I guess the answer is yes?)

Reply:
that makes sense to me.